R/prepare_pacta_dashboard_data.R

```diff
@@ -104,10 +104,6 @@ header_dictionary <- readr::read_csv(
   col_types = readr::cols()
 )
 
-js_translations <- jsonlite::fromJSON(
-  txt = system.file("extdata/translation/js_labels.json", package = "workflow.pacta.dashboard")
-)
-
```

Comment:
If these are not used anywhere in `workflow.pacta.dashboard` then we can remove these lines.

Reply:
yep, that's primarily what this PR does

Reply:
Haha, yeah, I can see that. I thought that it will be clearer when I comment on the lines that I agree that they can be removed. But apparently it was not.

Reply:
🤦🏻 misunderstood